shacl12-core/index.html

```diff
@@ -765,30 +765,80 @@ <h3>Document Conventions</h3>
 					A <a>shapes graph</a> is ill-formed if it contains at least one ill-formed node.
 				</p>
 			</section>
+			
+			<section id="shacl-rdfs">
+				<h3>Relationship between SHACL and RDFS inferencing</h3>
+				<p>
+					SHACL uses the RDF and RDFS vocabularies, but full RDFS inferencing is not required.
+				</p>
+				<p>
+					However, SHACL processors MAY operate on RDF graphs that include entailments [[!sparql12-entailment]],
+					whether pre-computed before being submitted to a SHACL processor or performed on the fly as
+					part of SHACL processing (without modifying either <a>data graph</a> or <a>shapes graph</a>).
+					To support processing of entailments, SHACL includes the property
+					<code>sh:entailment</code> to indicate the inferencing that is required
+					by a given <a>shapes graph</a>.
+				</p>
+				<p class="syntax">
+					<span data-syntax-rule="entailment-nodeKind">The <a>values</a> of the property <code>sh:entailment</code> are IRIs.</span>
+					Common values for this property are covered by [[!sparql12-entailment]].
+				</p>
+				<p>
+					SHACL implementations MAY, but are not required to, support entailment regimes.
+					If a <a>shapes graph</a> contains any <a>triple</a> with the <a>predicate</a> <code>sh:entailment</code> and <a>object</a> <code>E</code>
+					and the SHACL processor does not support <code>E</code> as an entailment regime for the given <a>data graph</a>,
+					then the processor MUST signal a <a>failure</a>.
+					Otherwise, the SHACL processor MUST provide the entailments for all of the values of <code>sh:entailment</code> in the <a>shapes graph</a>,
+					and any inferred triples MUST be returned by all queries against the <a>data graph</a> during the <a>validation</a> process.
+				</p>
+			</section>			
+		</section>
 
-			<section class="informative">
-				<h3>SHACL Example</h3>
+		<section id="getting-started" class="informative">
+   			<h2>Getting Started with SHACL Core</h2>
+			<p>
+				In this section, we will walk you through a simple example that introduces the basics of SHACL Core.
+				You will learn how to describe how your data should look, and how a SHACL processor checks whether your data meets that description.
```

Comment:
Sorry; I didn't get notification of the last change ... which I messed up. I'll make this suggestion as a comment, so the PR can be merged immediately after it's done.
```suggestion
				You will learn to describe how your data should look, and how a SHACL processor checks whether your data meets that description.
```